bigbone/src/test/kotlin/social/bigbone/api/method/PushNotificationMethodsTest.kt

```diff
@@ -0,0 +1,36 @@
+package social.bigbone.api.method
+
+import org.amshove.kluent.shouldBeEqualTo
+import org.amshove.kluent.shouldNotBeEqualTo
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import social.bigbone.api.exception.BigBoneRequestException
+import social.bigbone.testtool.MockClient
+
+class PushNotificationMethodsTest {
```

Comment:
Please add tests for the other methods. For the ones where we send data, I think it could make sense to verify that the payload sent is what is expected. This way, mistakes like the one you fixed in `StatusMethods` would hopefully not slip through.